swagger/v2/post_event_responses.yaml

```diff
@@ -23,6 +23,15 @@
       type: array
       items:
         $ref: "../v1/errors.yaml#/NotAuthorisedError"
+"404":
```

Comment:
can you add this new error to https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/spec/swagger/swagger_doc_v2.yaml#L689

Reply:
Done.